interactive-bots/helloworld/Symt/notes.md

```diff
@@ -0,0 +1,12 @@
+Notes:
+
+1. I came across an issue when trying to run:
+<pre><code>
```

Comment:
Use tripple backticks to enclose code blocks instead of pre.

Example:

```
` ` `
Code goes here. Place your backticks without spaces in them.
` ` `
```

Reply:
When i last used ` ` ` it didn't format